go.mod

```diff
@@ -316,6 +317,7 @@ replace (
 	github.com/google/gopacket => github.com/elastic/gopacket v1.1.20-0.20211202005954-d412fca7f83a
 	github.com/insomniacslk/dhcp => github.com/elastic/dhcp v0.0.0-20200227161230-57ec251c7eb3 // indirect
 	github.com/tonistiigi/fifo => github.com/containerd/fifo v0.0.0-20190816180239-bda0ff6ed73c
+	github.com/pierrec/lz4/v4 v4.1.15-0.20220523073431-885e900f46f2 => /Users/leehinman/src/pierrec_lz4
```

Comment:
The rename is necessary because the Flush function needs a small patch, it currently prevents further writing to the stream.  The patch allows you to write to the stream after a flush.  So before merging it would be good to see if upstream will take the fix or if we need to maintain a fork.

Reply:
The upstream project looks active enough, hopefully they'll just take the PR. I'm curious what the change here looks like as well.

Reply:
https://github.com/pierrec/lz4/pull/188

Reply:
Already merged! Nice, that was fast.